rq/worker.py

```diff
@@ -711,6 +717,16 @@ def heartbeat(self, timeout: Optional[int] = None, pipeline: Optional['Pipeline'
         connection.hset(self.key, 'last_heartbeat', utcformat(utcnow()))
         self.log.debug('Sent heartbeat to prevent worker timeout. Next one should arrive in %s seconds.', timeout)
 
+    def clean_batch_registries(self):
```

Comment:
`worker.py` is getting big, let's move this functionality into a `cleanup_batch_registry()` function in `batch.py`.

Reply:
Done